tools/tcoffee/test/main.nf

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nextflow
+
+nextflow.preview.dsl = 2
+
+include check_output from  '../../../nf-core/module_testing/check_process_outputs.nf'
+include tcoffee from '../main.nf'
+
+// Define input channels
+fasta = Channel.fromPath('../../../test-datasets/tools/tcoffee/input/BBA0001.tfa')
+
+// Run the workflow
+workflow {
+    tcoffee(fasta)
+    // .check_output()
```

Comment:
I guess the commented line can be removed.

Reply:
I left it since as far as I understood, the `check_output` module will be used to check any of the outputs of the other modules, am I right @ewels ?

Reply:
Yes, hopefully we can add some generic checking functions for this kind of thing soon